### PR TITLE
Governance stage 1

### PR DESCRIPTION
**Pull request by MylesBorins**
This PR relies on https://github.com/nodejs/bootstrap/pull/23

It moves the GOVERNANCE proposal to Stage 1. In this stage we are agreeing that the draft of the proposal is written in the language that will be officially adopted but that we need to reach consensus on the implementation.

The proposal also clearly outlines the expected steps to get to both stage 2 and stage 3. Please note that this proposal is an example of one that I believe can skip stage 2 and immediately go to stage three as there is not a need for board, legal, or budget approval.

For this PR please refer only to 78df381 when approving

### CODE REVIEW

File: proposals/TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+A rough description of the program idea or policy
+
+## Required Resources
+
+What resources would be needed to execute it?
+
+## Who would be responsible?
+
+Which committee would be overse
```
Comment (mcollina) [inline proposals/TEMPLATE.md]: This is a truncated sentence.

File: proposals/TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
+# My New Proposal
+>  Stage 0
+
+Below is simply a suggested template for drafting a proposal, it will not fit all potential proposals.
+
+## Champion
+
+Here is where you identify stakeholders
```
Comment (mcollina) [inline proposals/TEMPLATE.md]: missing . at the end

File: proposals/README.md [hunk 3]
```diff
@@ -1,5 +1,19 @@
...
-* [MylesBorins - GOVERNANCE](mylesborins-GOVERNANCE.md)
+There are currently no Stage 0 proposals
+
+## Stage 1
+
+* [MylesBorins - GOVERNANCE](mylesborins-GOVERNANCE)
+
+## Stage 2
+
+There are currently no Stage 2 proposals
```
Comment (mcollina) [inline proposals/README.md]: missing . at the end

File: proposals/README.md [hunk 4]
```diff
@@ -1,5 +1,19 @@
...
+
+## Stage 3
+
+There are currently no Stage 3 proposals
```
Comment (mcollina) [inline proposals/README.md]: missing . at the end

File: proposals/mylesborins-GOVERNANCE/README.md [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+
+Myles Borins (@Myles Borins)
+
+## Description
+
+This document is a basic outline of the governance model for the bootstrap team. It outline how membership is defined and how we will collaborate.
+
+## Required Resources
+
+Approval of the bootstrap team
```
Comment (mcollina) [inline proposals/mylesborins-GOVERNANCE/README.md]: . missing

File: README.md [hunk 6]
```diff
@@ -1,2 +1,7 @@
 # bootstrap
+
 Foundation Bootstrap Team
+
+## Making proposals
+
+Anyone may submit an idea for a policy or program following the [staging process](STAGING_PROCESS.md)
```
Comment (mcollina) [inline README.md]: Can you close this sentence with a .?

Should we link this to CONTRIBUTING.md as well?

File: proposals/TEMPLATE.md [hunk 7]
```diff
@@ -0,0 +1,32 @@
+# My New Proposal
```
Comment (amiller-gh) [inline proposals/TEMPLATE.md]: I think missing from this is sections for **Drawbacks**, **Alternatives**, and **Unresolved Questions**. Prompting the drafter to also lay out the counter argument can inspire more productive conversation.

**Pull request merged**